aws-ssm-parameter/src/test/java/com/amazonaws/ssm/parameter/CreateHandlerTest.java

```diff
@@ -0,0 +1,275 @@
+package com.amazonaws.ssm.parameter;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import software.amazon.awssdk.services.ssm.SsmClient;
+import software.amazon.awssdk.services.ssm.model.GetParametersResponse;
+import software.amazon.awssdk.services.ssm.model.GetParametersRequest;
+import software.amazon.awssdk.services.ssm.model.Parameter;
+import software.amazon.awssdk.services.ssm.model.ParameterNotFoundException;
+import software.amazon.awssdk.services.ssm.model.PutParameterResponse;
+import software.amazon.awssdk.services.ssm.model.PutParameterRequest;
+import software.amazon.awssdk.services.ssm.model.ParameterTier;
+import software.amazon.awssdk.services.ssm.model.TooManyUpdatesException;
+import software.amazon.cloudformation.exceptions.CfnNotFoundException;
+import software.amazon.cloudformation.exceptions.CfnThrottlingException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.OperationStatus;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class CreateHandlerTest extends AbstractTestBase {
+
+    @Mock
+    private AmazonWebServicesClientProxy proxy;
+
+    @Mock
+    private ProxyClient<SsmClient> proxySsmClient;
+
+    @Mock
+    SsmClient ssmClient;
+
+    private CreateHandler handler;
+
+    private ResourceModel RESOURCE_MODEL;
+
+    @BeforeEach
+    public void setup() {
+        handler = new CreateHandler();
+        ssmClient = mock(SsmClient.class);
+        proxy = new AmazonWebServicesClientProxy(logger, MOCK_CREDENTIALS, () -> Duration.ofSeconds(600).toMillis());
+        proxySsmClient = MOCK_PROXY(proxy, ssmClient);
+
+        RESOURCE_MODEL = ResourceModel.builder()
+                .description(DESCRIPTION)
+                .name(NAME)
+                .value(VALUE)
+                .type(TYPE)
+                .tags(TAG_SET)
+                .build();
+    }
+
+    @Test
```

Comment:
plz add `@AfterEach` to verify no more interactions
[example](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-kms/blob/master/key/src/test/java/software/amazon/kms/key/CreateHandlerTest.java#L63-L67)

Reply:
Done